mypy/stubtest.py

```diff
@@ -366,10 +370,7 @@ def _belongs_to_runtime(r: types.ModuleType, attr: str) -> bool:
         else {
             m
             for m in dir(runtime)
-            if not is_probably_private(m)
-            # Ensure that the object's module is `runtime`, since in the absence of __all__ we
```

Comment:
This comment still seems accurate and helpful

Reply:
My thinking was that it isn't the only heuristic we use anymore, so imo it's not accurate — we don't _always_ ensure that the object's module is `runtime`. It is _sometimes_ still the heuristic we use, which is why I added the replacement comment on line 354.

Reply:
I meant the comment literally; that is, "object's module" not "object's __module__" ;-)

Maybe it needs some rephrasing, but I think there's a) value in saying this is a guess because we don't have __all__, b) mentioning re-exports explicitly because they're an important failure case.

Reply:
Ahh I see! I'll add it back and see if I can clarify, in that case